core/serialize/serialize.cc

```diff
@@ -73,15 +72,15 @@ vector<u1> Pickler::result(int compressionDegree) {
         data.emplace_back(zeroCounter);
         zeroCounter = 0;
     }
-    const size_t maxDstSize = Lizard_compressBound(data.size());
+    const size_t maxDstSize = LZ4_compressBound(data.size());
     vector<u1> compressedData;
     compressedData.resize(2048 + maxDstSize); // give extra room for compression
                                               // Lizard_compressBound returns size of data if compression
                                               // succeeds. It seems to be written for big inputs
                                               // and returns too small sizes for small inputs,
                                               // where compressed size is bigger than original size
-    int resultCode = Lizard_compress((const char *)data.data(), (char *)(compressedData.data() + SIZE_BYTES * 2),
-                                     data.size(), (compressedData.size() - SIZE_BYTES * 2), compressionDegree);
```

Comment:
I guess we no longer use `compressionDegree`?  Is it possible to change that, so we are explicitly selecting the compression settings LZ4 will use?

Reply:
Yeah, we're using the default. I can make it explicit?

Reply:
I am a fan of explicit so some future upgrade doesn't silently change the defaults out from under us.  Please change.